### DISCUSSION TITLE
Is `useToggle` obsolete?

### DISCUSSION THREAD

**Discussion by k-melnychuk** [juliencrn/usehooks-ts#548] [General]
Hey @juliencrn,
Just wanted to say thanks for such a great repo!
I’ve opened a couple of pull requests in the past and happy to contribute in the future. It’s definitely my go-to module for react utility hooks.

I’ve noticed that `useBolean` offers the same functionality as `useToggle` and even more. Does that make `useToggle` obsolete and maybe it’s worth deprecating it?

**Comment by juliencrn** [comment]
Hi @k-melnychuk, 

Thank you! ❤️🚀

`useBoolean` and `useToggle` are indeed pretty similar, some people seem to prefer one API over the other. I'm open, but removing one includes a breaking change. Idk tbh. 

Community: Feel free to give your advice!
